build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
You might need to put `3.10` in quotes (i.e. `'3.10'`) since the yaml parser may interpret it as a number and in that case `3.10` == `3.1`